action.yaml

```diff
@@ -149,6 +149,13 @@ runs:
         WORKSPACE: ${{ env.workspace }}
         BUILD_URL: ${{ env.build_url }}
 
+    - name: Create and Update frogbot config file
+      if: ${{ env.Changed_module }}
```

Comment:
With this condition alone, this step will be run every time there is a change detection file list. We want to be run when the jfrog bot step is being called. So, the condition has to be when JF_URL is not null and there are change modules.

Reply:
Since it is applicable only for maven project ,In workflow file we need to add an extra step for creating the config file on fly so for creating that we added the above condition to execute the composite action only when changed module is passed as env variable .

Reply:
So, if we have a maven project but the workflow does not call the jfrog bot step, what will happen? Will the jfrog bot config be created anyway or not?

Reply:
if: ${{ env.Changed_module }} when we passed these condition then only the config will be created

Reply:
Again, consider this scenario:

if we have a maven project and in the workflow that builds it, we use change detection **but** it does not call the jfrog bot step.

In this scenario what will happen? Will the jfrog bot config be created anyway or not?